src/ray/stats/metric_exporter_client.cc

```diff
@@ -50,36 +57,109 @@ MetricsAgentExporter::MetricsAgentExporter(std::shared_ptr<MetricExporterClient>
   client_.reset(new rpc::MetricsAgentClient(address, port, client_call_manager_));
 }
 
-void MetricsAgentExporter::ReportMetrics(const std::vector<MetricPoint> &points) {
-  MetricExporterDecorator::ReportMetrics(points);
-  rpc::ReportMetricsRequest request;
-  for (auto point : points) {
-    auto metric_point = request.add_metrics_points();
-    metric_point->set_metric_name(point.metric_name);
-    metric_point->set_timestamp(point.timestamp);
-    metric_point->set_value(point.value);
-    auto mutable_tags = metric_point->mutable_tags();
-    for (auto &tag : point.tags) {
-      (*mutable_tags)[tag.first] = tag.second;
-    }
-    // If description and units information is requested from
-    // the metrics agent, append the information.
-    // TODO(sang): It can be inefficient if there are lots of new registered metrics.
-    // We should make it more efficient if there's compelling use cases.
-    if (should_update_description_) {
-      metric_point->set_description(point.measure_descriptor.description());
-      metric_point->set_units(point.measure_descriptor.units());
+void MetricsAgentExporter::ReportMetrics(
+    const std::vector<MetricPoint> &points,
```

Comment:
Can we just make it only receive the second argument? I think if @ashione wants to process metrics, he can write his own client that translates opencensus data into MetricPoint.